java/src/jmri/server/json/operations/JsonOperationsHttpService.java

```diff
@@ -155,7 +155,7 @@ public JsonNode doPost(String type, String name, JsonNode data, Locale locale, i
 
     @Override
     // Nullable overrides super class'es non-null requirement for name
-    public JsonNode doPut(String type, @Nullable String name, JsonNode data, Locale locale, int id) throws JsonException {
+    public JsonNode doPut(String type, @CheckForNull String name, JsonNode data, Locale locale, int id) throws JsonException {
```

Comment:
This must be Nullable per https://static.javadoc.io/com.google.code.findbugs/jsr305/3.0.2/javax/annotation/Nullable.html and http://findbugs.sourceforge.net/manual/annotations.html since this is overriding a Nonnull annotation, and is called out for use in that manner by those documents.

Reply:
The linked docs are also consistent with using @CheckForNull here: "Use CheckForNull to indicate that the element value should always be checked for a null value."